spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/security/IapAuthenticationAutoConfiguration.java

```diff
@@ -54,13 +56,14 @@
 @AutoConfigureBefore(OAuth2ResourceServerAutoConfiguration.class)
 public class IapAuthenticationAutoConfiguration {
 
-	// TODO: externalize as properties.
-	private static final String PUBLIC_KEY_VERIFICATION_LINK = "https://www.gstatic.com/iap/verify/public_key-jwk";
-
-	private static final String ENCRYPTION_ALGORITHM = "ES256";
+	@VisibleForTesting
```

Comment:
What is the problem to make all these constants `public`?
to be honest I'm fully against such a `@VisibleForTesting` hack in the production code.
This is not Spring way...

Reply:
Makes sense. It's a cultural difference; I am still adjusting -- in a giant monorepo, a convention to restrict visibility, unless there is a good reason to relax it, is reasonable. But in Spring, where every project's boundaries are clearly delineated, it makes sense to do the opposite.